app/views/layouts/_card_view.html.erb

```diff
@@ -12,6 +12,5 @@
     <%else%> 
         Not deliverable
     <%end%>
-    <%= item.category %>
```

Comment:
why was this removed?

Reply:
honestly I removed it coz it was taking a lot of space, then I fixed the size of the container instead but didn't put the category back. also I felt like doesn't really matter much

Reply:
if it is a clean-look vs. category showing tradeoff, then I would leave it as is

otherwise, category might be nice